pkg/crc/validation/validation.go

```diff
@@ -75,6 +76,16 @@ func ValidateBundlePath(bundle string) error {
 	return nil
 }
 
+func ValidateBundle(bundlePath string) error {
+	bundleName := filepath.Base(bundlePath)
+	_, err := bundle.GetCachedBundleInfo(bundleName)
```

Comment:
This check is just a feel good one because if it error occur we are going to run `ValidateBundlePath` so I am not able to figure out what advantage it is going to add.

Reply:
With this change, if for some reason `~/.crc/cache/crc_libvirt_4.6.9` exists but `~/.crc/cache/crc_libvirt_4.6.9.crcbundle` does not exist, we can still successfully run `crc start`. With current crc master, `~/.crc/cache/crc_libvirt_4.6.9.crcbundle` must exist even if the uncompressed bundle is present.